src/presentation/http/router/note.test.ts

```diff
@@ -330,53 +399,31 @@ describe('Note API', () => {
     });
 
     test.todo('Returns 400 when parentId has incorrect characters and length');
+  });
 
-    test('Returns 200 when user is a team member with a Write role', async () => {
-      /** Create test user - creator of a note */
-      const creator = await global.db.insertUser();
-
-      /** Create test user */
+  describe('POST /note', () => {
+    test('Should correctly save relation to parent note if parentId passed', async () => {
       const user = await global.db.insertUser();
 
-      /** Create test note */
-      const note = await global.db.insertNote({
-        creatorId: creator.id,
+      const parentNote = await global.db.insertNote({
+        creatorId: user.id,
       });
 
-      /** Create test note settings */
       await global.db.insertNoteSetting({
-        noteId: note.id,
+        noteId: parentNote.id,
         isPublic: true,
       });
 
-      /** Create test team */
-      await global.db.insertNoteTeam({
-        noteId: note.id,
-        userId: user.id,
-        role: MemberRole.Write,
-      });
-
       const accessToken = global.auth(user.id);
 
       let response = await global.api?.fakeRequest({
-        method: 'PATCH',
+        method: 'POST',
         headers: {
           authorization: `Bearer ${accessToken}`,
         },
-        url: `/note/${note.publicId}`,
+        url: `/note`,
         body: {
-          content: {
-            blocks: [
-              {
-                id: 'qxnjUh9muR',
-                type: 'header',
-                data: {
-                  text: 'sample text',
-                  level: 1,
-                },
-              },
-            ],
-          },
+          parentId: parentNote.publicId,
```

Comment:
Can't we have an empty note?

Reply:
@neSpecc

Reply:
no, there is always at least `content: { blocks: [] }`

Reply:
Isn't that being handled in the route?

![image](https://github.com/codex-team/notes.api/assets/59017579/411d97cf-1544-4176-8088-d417dbf84387)

Reply:
I don't know a case when Client can send note without a content. It will always send a content
I think we need to mark this field as `required` via schema.

Reply:
got it